src/main/java/org/folio/marccat/resources/AuthorityRecordAPI.java

```diff
@@ -63,4 +67,20 @@
 
   }
 
+  @GetMapping("/authority-record/from-template/{idTemplate}")
+  public AuthorityRecord getEmptyRecord(@PathVariable final Integer idTemplate, @RequestParam final String lang,
```

Comment:
You can format with one annotation per line?

Reply:
I have changed it